sdk/authorization/Azure.ResourceManager.Authorization/src/autorest.md

```diff
@@ -113,7 +113,7 @@ generate-arm-resource-extensions:
   - /{scope}/providers/Microsoft.Authorization/roleEligibilityScheduleInstances/{roleEligibilityScheduleInstanceName}
 
 request-path-to-resource-type:
-  /{scope}/providers/Microsoft.Authorization/roleManagementPolicyAssignments/{roleManagementPolicyAssignmentName}: Microsoft.Authorization/roleManagementPolicyAssignment
+  /{scope}/providers/Microsoft.Authorization/roleManagementPolicyAssignments/{roleManagementPolicyAssignmentName}: Microsoft.Authorization/roleManagementPolicyAssignments
```

Comment:
well if we just remove this line, the resource type is fixed.
Why we have this config in the first place?